server/server.go

```diff
@@ -142,6 +142,7 @@ func (s *Server) StartContext(ctx context.Context, host, port string) error {
 	}
 	l, err := s.listener(host, port)
 	if err != nil {
+		s.Errorf("Cannot listen: %w", err)
```

Comment:
how does this cause it fail gracefully? this just prints a message doesnt it...

Reply:
It seems when I originally submitted this PR, my change in server_listen.go fixed an actual issue which you also fixed two weeks later in response to https://github.com/jpillora/chisel/issues/232 so this PR is no longer valid

today I checked the merge conflict on this one, resolved it but didn't realize this was fixed elsewhere